libpod/runtime_pod.go

```diff
@@ -181,3 +181,27 @@ func (r *Runtime) GetRunningPods() ([]*Pod, error) {
 	}
 	return runningPods, nil
 }
+
+// GetPodsByContext gets pods whether all, latest, or a slice of names/ids
+func (r *Runtime) GetPodsByContext(all, latest bool, pods []string) ([]*Pod, error) {
```

Comment:
Can we avoid adding this? I really don't want to overpopulate libpod with endpoints to add containers/pods. This one seems to belong in cmd/podman/shared or libpod/adapter - it's very much a podman specific thing

Reply:
moved to adapter/shortcuts